base/audit_events.go

```diff
@@ -814,6 +813,9 @@ var AuditEvents = events{
 			AuditFieldFeedType:    "continuous, normal, longpoll, websocket, etc.",
 			AuditFieldIncludeDocs: true,
 		},
+		optionalFieldGroups: []fieldGroup{
+			fieldGroupRequest, // active replicator push doesn't have local and remote
```

Comment:
Do we want to audit usage by the active push replicator?

Reply:
We decided not to do this.